docs/ad/security-ad.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Anomaly Detection Security
+nav_order: 10
+parent: Anomaly Detection
+has_children: false
+---
+
+# Anomaly detection security
+
+You can use the security plugin with anomaly detection to limit non-admin users to specific actions. For example, you might want some users to only be able to create and view detectors, while others to modify and delete detectors.
```

Comment:
while others to modify

I think you've got a typo here.

Reply:
Is it? It's you want some users to ---, while others to ---.

Reply:
Creating is also kind of modifying, may be will that confuse customers?
May be what we can say is 'some users might want to create/update/delete detectors where as some of them only want to view detectors and their results'

Reply:
Done.